src/main/java/uk/gov/ons/validation/entity/WranglerResponseData.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.ons.validation.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class WranglerResponseData {
+    public String primaryValue ;
```

Comment:
Should have a blank line here and remove the excess lines at the end of this class.